src/ui/components/Dashboard/DashboardViewer.tsx

```diff
@@ -117,11 +119,23 @@ export default function DashboardViewer({ recordings }: { recordings: Recording[
               value={searchString}
               onChange={e => setSearchString(e.target.value)}
             />
+            <SelectMenu
+              selected={limit}
+              setSelected={value => setLimit(value as Limit)}
+              options={[
+                { id: "all", name: "Show all" },
+                { id: "20", name: "20" },
+                { id: "50", name: "50" },
+                { id: "100", name: "100" },
+              ]}
+              className="w-32"
+            />
```

Comment:
I'd probably leave this out for now because it shouldn't be as primary as the other dropdowns